### PR TITLE
chore: sonar cloud reduce

### PR DESCRIPTION
**Pull request by bashleigh**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #

### CODE REVIEW

File: packages/admin-portal/src/actions/customers.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { actionCreator } from '@/utils/actions'
 import ActionTypes from '@/constants/action-types'
-import { PagedResultCustomerModel_ } from '@/services/customers'
+import { PagedResultCustomerModel } from '@/services/customers'
```
Comment (willmcvay) [inline packages/admin-portal/src/actions/customers.ts:3]: Can be exported from TS-Definitions right?

File: packages/admin-portal/src/components/ui/app-revision-comparison/app-revision-comparison.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import DiffMedia from '@/components/ui/diff-media'
 import DiffCheckbox from '../diff-checkbox'
 import DiffViewer from '../diff-viewer'
 import DiffRenderHTML from '../diff-render-html'
-import { DesktopIntegrationTypeModel, PagedResultDesktopIntegrationTypeModel_ } from '@/types/desktop-integration-types'
+import { DesktopIntegrationTypeModel, PagedResultDesktopIntegrationTypeModel } from '@/types/desktop-integration-types'
```
Comment (willmcvay) [inline packages/admin-portal/src/components/ui/app-revision-comparison/app-revision-comparison.tsx:8]: Ditto, not sure why they are not coming from TS-definitions, bit weird that

File: packages/admin-portal/src/reducers/apps/revisions.ts [hunk 3]
```diff
@@ -8,13 +8,13 @@ import {
   setRequestDeclineRevisionFormState,
   fetchRevision,
 } from '@/actions/revision-detail'
-import { PagedResultDesktopIntegrationTypeModel_ } from '@/types/desktop-integration-types'
+import { PagedResultDesktopIntegrationTypeModel } from '@/types/desktop-integration-types'
```
Comment (willmcvay) [inline packages/admin-portal/src/reducers/apps/revisions.ts:11]: And again!

File: packages/admin-portal/src/sagas/apps/__stubs__/integration-types.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
-import { PagedResultDesktopIntegrationTypeModel_ } from '@/types/desktop-integration-types'
+import { PagedResultDesktopIntegrationTypeModel } from '@/types/desktop-integration-types'
```
Comment (willmcvay) [inline packages/admin-portal/src/sagas/apps/__stubs__/integration-types.ts:1]: And again

**Review by willmcvay — APPROVED** [review]
Am approving because we are re-writing anyway and it's just some small stuff but if you can easily fix, worth doing

File: packages/admin-portal/src/sagas/apps/__stubs__/integration-types.ts (hunk 4 above)
Comment (bashleigh) [reply packages/admin-portal/src/sagas/apps/__stubs__/integration-types.ts:1]: I thought so too but decided against it because of the rewrite

**Pull request merged**